app/user/user.api.ts

```diff
@@ -0,0 +1,49 @@
+import { injectable } from '@servicetitan/react-ioc';
+import { AxiosPromise } from 'axios';
+
+import { mockResponse } from '../mocks/api';
+import { Dave, Users } from '../mocks/user';
+
+export interface UserDTO {
+    id: number;
+    login: string;
+    role: string;
+}
+
+export type EditableUserDTO = Pick<UserDTO, Exclude<keyof UserDTO, 'id'>>;
```

Comment:
I'm sure that we won't create separate DTO's for creating and edit. It so useful for us, but not looks real.